Source/x11/XGServerWindow.m

```diff
@@ -576,7 +576,7 @@ - (NSRect) _XFrameToOSFrame: (NSRect)x for: (void*)window
 
   [self styleoffsets: &l : &r : &t : &b : style : win->ident];
   o = x;
-  o.origin.y = [self boundsForScreen: win->screen].size.height - x.origin.y;
+  o.origin.y = [self boundsForScreen: win->monitor_id].size.height - x.origin.y;
```

Comment:
Could you explain why you use the height of the monitor here while you used the screen height in _OSFrameToXFrame: These two methods should be symmetrical.

Reply:
Thanks, fixed.